src/CurrentRoute.php

```diff
@@ -25,6 +25,16 @@ final class CurrentRoute
      */
     private ?UriInterface $uri = null;
 
+    /**
+     * Returns the current Route name
+     *
+     * @return string
+     */
+    public function getName(): string
+    {
+        return $this->route === null ? '' : $this->route->getName();
```

Comment:
Maybe better return `?string`?

My logic:
_empty string_ — route exist, but empty.
`null` — route not exist.

Reply:
Agree. The code was moved from demo layout :)